modules/core/arrow-core-data/src/main/kotlin/arrow/typeclasses/Bitraverse.kt

```diff
@@ -39,7 +39,6 @@ import arrow.typeclasses.internal.IdBimonad
  * ## Bisequence
  *
  * `Bisequence` invert the original structure `F<G<A,B>>` to `G<F<A>,F<B>>`
- * `Bisequence` invert the original structure `F<G<A,B>>` to `G<F<A>,F<B>>`
```

Comment:
Bitraverse was from your PR right?

Reply:
yes, I did last week, but I have realised I put this duplicate line